server/services/educationAndWorkPlanService.ts

```diff
@@ -73,4 +76,28 @@ export default class EducationAndWorkPlanService {
       return new Map()
     }
   }
+
+  async getEducation(prisonNumber: string, token: string): Promise<EducationResponse> {
+    try {
+      const educationResponse = await this.educationAndWorkPlanClient.getEducationResponse(prisonNumber, token)
+      return toCreateEducationRequest(educationResponse)
+    } catch (error) {
```

Comment:
I'm not sure about this catch block ... I'll need to give it some thought 🤔 ....

I know it's a pattern we have in other similar places, but I'm not sure how well it plays with the catch block and `gracefullyHandleException` function in the middleware you've written 🤔  (ie. if the service method catches and handles the exception, then the exception can never been seen/handled by the middleware ....

I'll need to give this a little thought .... it might be indicative of problems we have elsewhere .... 🤔

Reply:
Have left as is for now.